lib/phlex/sgml.rb

```diff
@@ -304,5 +304,14 @@ def capture(&block)
 
 			buffer
 		end
+
+		# This should be the last method defined
+		def self.method_added(method_name)
```

Comment:
@joeldrapper I didn't know if we wanted to move this guardrail to `SGML`, or duplicate it across `HTML` and `SVG`. Due to the strategy change, it is inheritable. But inheriting it also means that every method defined in `HTML` and `SVG` are needlessly incurring the penalty of checking this guardrail. But maybe the one time performance cost is worth the extra dryness.

Reply:
Shall we put it in a module and have HTML and SVG extend that module as the last thing?